bofire/data_models/surrogates/mixed_single_task_gp.py

```diff
@@ -100,27 +105,81 @@ def _update_hyperparameters(
 class MixedSingleTaskGPSurrogate(TrainableBotorchSurrogate):
     type: Literal["MixedSingleTaskGPSurrogate"] = "MixedSingleTaskGPSurrogate"
     continuous_kernel: AnyContinuousKernel = Field(
-        default_factory=lambda: MaternKernel(
-            ard=True, nu=2.5, lengthscale_prior=HVARFNER_LENGTHSCALE_PRIOR()
+        default_factory=lambda: RBFKernel(
```

Comment:
Is there a reason why the default changed?

Reply:
This is now the same default that also botorch has.